service/controller/resource/statuscondition/create.go

```diff
@@ -106,11 +106,12 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 }
 
 func (r *Resource) computeCreateClusterStatusConditions(ctx context.Context, cr infrastructurev1alpha2.CommonClusterObject, nodes []corev1.Node, machineDeployments []apiv1alpha2.MachineDeployment) error {
-	cc, err := controllercontext.FromContext(ctx)
+	componentVersions, err := r.releaseVersion.ComponentVersion(ctx, cr)
 	if err != nil {
 		return microerror.Mask(err)
 	}
 
+	providerOperator := fmt.Sprintf("%s-operator", r.provider)
 	providerOperatorVersionLabel := fmt.Sprintf("%s-operator.giantswarm.io/version", r.provider)
```

Comment:
I think the label is now `aws-operator-operator.giantswarm.io/version`, which is wrong.

Reply:
But isn't the provider label "kvm", "aws" or "azure"? without "-operator"

Reply:
Right, I think I confused `providerOperator` and `r.provider`. So this should be good then. 🙈